### ISSUE TITLE
iPAS AI 應用規劃師科目3練習 - 2026-09-14

### ISSUE THREAD

**Issue by JohnnyLai92** [JohnnyLai92/ipas#20]
# iPAS AI 應用規劃師科目3練習 - 2026-09-14（第 10 輪）

本次共 **10 題**，側重 **L232 機器學習與深度學習**，次要 **L233 建模與參數調校**，並以其他領域補齊跨域涵蓋。情境以「醫療與健康照護」與「製造與工業檢測」兩種產業為背景。

**作答方式**：請**直接點選下方每個選項前的核取方塊**即可作答，不需留言、不需進入編輯畫面。單選題請只勾一個；多選題請依標示勾選指定數量；排序／流程題請勾出「正確順序」那一個選項。

**批改方式**：全部勾選完成後，請將本 issue 的 label 改為 `ai-analysis`，系統即會進行批改與錯題分析。

## 第 1 題〔單選〕L23201 機器學習原理

某醫院訓練一個預測「住院天數」的模型。團隊發現：**訓練集準確度只有 68%，驗證集準確度也只有 66%**，兩者都偏低且差距很小。下列對此現象的判斷與對策，何者最正確？

- [ ] (A) 屬於過擬合（Overfitting），應加入更強的正則化或 Dropout
- [ ] (B) 屬於欠擬合（Underfitting，高偏差），應增加模型複雜度或補入更有效的特徵
- [ ] (C) 屬於資料外洩（Data Leakage），應重新切分訓練與測試集
- [ ] (D) 模型已達最佳狀態，兩者差距小代表泛化良好，無須調整

---

## 第 2 題〔單選〕L23202 常見機器學習演算法

某工廠要用「溫度、振動、電流」等感測數值判斷機台是否即將故障（二元分類）。稽核單位要求模型**能明確畫出 if-then 決策規則**向現場人員解釋判斷依據，同時仍希望有一定的準確度。下列模型何者**最符合「內在可解釋、可呈現決策規則」**的需求？

- [ ] (A) 使用 RBF 核的 SVM
- [ ] (B) 深度神經網路（DNN）
- [ ] (C) 決策樹（Decision Tree）
- [ ] (D) KNN（K 最近鄰）

---

## 第 3 題〔單選 · 程式判讀〕L23203 深度學習原理與框架

某製造檢測團隊以 PyTorch 串接兩個全連接層處理一批影像特徵向量：

```python
import torch
import torch.nn as nn

layer1 = nn.Linear(16, 8)
layer2 = nn.Linear(8, 3)

x = torch.randn(4, 16)   # batch=4
h = layer1(x)
y = layer2(h)
print(y.shape)
```

執行後 `print(y.shape)` 的輸出為何？

- [ ] (A) torch.Size([4, 16])
- [ ] (B) torch.Size([4, 8])
- [ ] (C) torch.Size([4, 3])
- [ ] (D) torch.Size([8, 3])

---

## 第 4 題〔多選（可勾選 3 項）〕L23203 深度學習原理與框架

下列關於深度學習模型與元件的敘述，哪些**正確**？（請勾選 3 項）

- [ ] (A) CNN 透過卷積運算擷取空間局部特徵，常用於影像辨識任務
- [ ] (B) LSTM 以閘門機制緩解梯度消失，較一般 RNN 更能捕捉長期序列依賴
- [ ] (C) Softmax 常用於多分類輸出層，會把輸出轉為總和為 1 的機率分布
- [ ] (D) ReLU 的作用是把輸入壓縮到 0~1 之間，因此常用於二分類的機率輸出
- [ ] (E) 只要不斷加大 Batch Size，模型的測試準確度必定隨之提升

---

## 第 5 題〔排序／流程 · 單選〕L23203 深度學習原理與框架

某製造瑕疵分類的神經網路，在一次小批次（mini-batch）訓練中會經歷下列 4 個步驟。請選出**正確的執行順序**：

① 依學習率更新權重與偏差
② 前向傳播（Forward Propagation）計算預測輸出
③ 反向傳播（Backpropagation）計算各參數的梯度
④ 以損失函數計算預測與真實標籤的誤差

- [ ] (A) ② → ④ → ③ → ①
- [ ] (B) ② → ③ → ④ → ①
- [ ] (C) ④ → ② → ③ → ①
- [ ] (D) ① → ② → ④ → ③

---

## 第 6 題〔單選〕L23303 模型評估指標

某醫院導入一套「早期癌症篩檢」模型，篩檢陽性者會再轉介進一步檢查。院方最擔心的是**把真正罹病的患者漏判為健康**（漏診）。在資源允許複檢的前提下，模型調校時應**優先提高**下列哪一項指標？

- [ ] (A) Precision（精確率）
- [ ] (B) Recall（召回率／敏感度）
- [ ] (C) 整體 Accuracy（準確率）
- [ ] (D) 訓練集的損失值（Training Loss）

---

## 第 7 題〔排序／流程 · 單選〕L23301 數據準備與特徵工程

某製造團隊要對含數值特徵的資料做標準化（StandardScaler）並訓練模型。為**避免資料外洩（Data Leakage）**，下列步驟的正確順序為何？

① 對訓練集與測試集分別套用（transform）已學到的縮放參數
② 先切分訓練集與測試集
③ 完成資料清理與缺失值處理
④ 只用「訓練集」計算（fit）標準化的平均數與標準差
⑤ 以縮放後的訓練資料訓練模型

- [ ] (A) ③ → ② → ④ → ① → ⑤
- [ ] (B) ③ → ④ → ② → ① → ⑤
- [ ] (C) ④ → ① → ③ → ② → ⑤
- [ ] (D) ② → ③ → ⑤ → ④ → ①

---

## 第 8 題〔多選（可勾選 3 項）〕L23302 模型訓練與調校

某醫療影像模型出現**訓練準確度 98%、驗證準確度 74%**的明顯過擬合。下列哪些做法**有助於改善過擬合**？（請勾選 3 項）

- [ ] (A) 加入 L2 正則化或 Dropout
- [ ] (B) 使用 Early Stopping，在驗證表現變差時提前停止訓練
- [ ] (C) 增加訓練資料量或使用資料擴增（Data Augmentation）
- [ ] (D) 移除驗證集，只用訓練集評估以拉高整體數字
- [ ] (E) 大幅增加模型層數與參數量，讓模型更完美擬合訓練資料

---

## 第 9 題〔單選〕L23103 數值優化技術與方法

某製造團隊以梯度下降訓練模型時，發現 **Loss 在訓練過程中劇烈震盪、甚至不斷變大而無法收斂**。在其他設定正常的情況下，下列何者是**最可能的原因與對策**？

- [ ] (A) 學習率（Learning Rate）設得過高，應調小學習率
- [ ] (B) 學習率設得過低，應大幅調高學習率
- [ ] (C) 訓練資料太多，應刪除大部分訓練樣本
- [ ] (D) 模型參數已收斂到全域最小值，屬正常現象無須處理

---

## 第 10 題〔情境分析 · 單選〕L23402 演算法偏見與公平性

某醫院的疾病風險模型上線後，發現對**兩個條件相近但性別不同的族群，核定的高風險比例差異極大**。下列處理方式何者最為恰當？

- [ ] (A) 只要整體 Accuracy 夠高即可，族群差異不需理會
- [ ] (B) 檢查訓練資料與特徵是否存在偏差，並以公平性指標評估各族群表現，必要時重新調整或訓練模型
- [ ] (C) 直接刪除測試資料中表現較差的族群樣本，讓數字變好看
- [ ] (D) 增加神經網路層數即可自動消除偏見，無須檢視資料
